content/hi/docs/reference/glossary/cluster-architect.md

```diff
@@ -0,0 +1,17 @@
+---
+title: क्लस्टर आर्किटेक्ट 
+id: cluster-architect
+date: 2018-04-12
+full_link: 
+short_description: >
+ एक व्यक्ति जो बुनियादी ढांचे को रचना करता है जिसमें एक या एक से अधिक कुबेरनेट्स क्लस्टर शामिल होते हैं।
+ 
+aka: 
+tags:
+- user-type
+---
+एक व्यक्ति जो बुनियादी ढांचे को रचना करता है जिसमें एक या एक से अधिक कुबेरनेट्स क्लस्टर शामिल होते हैं।
+
+<!--more-->
+
+क्लस्टर आर्किटेक्ट्स से उच्च उपलब्धता और सुरक्षा जैसे वितरित सिस्टम के साथ सर्वोत्तम अभ्यास अनुभव होने की उम्मीद है।

```

Comment:
Can this line be localized in a better way something along the lines of this?:
```suggestion
क्लस्टर आर्किटेक्ट्स वितरित सिस्टम के लिए सर्वोत्तम प्रथाओं से संबंधित हैं, उदाहरण के लिए उच्च उपलब्धता और सुरक्षा।
```
Thoughts, WDYT @bishal7679 @NitishKumar06 @Shivaniiiiiiiiiii ?

Reply:
@bishal7679 now check this

Reply:
@Babapool good suggestion but localize system to प्रणाली
👍

Reply:
@bishal7679  should I change सिस्टम to प्रणाली ?

Reply:
Yes

Reply:
@Babapool Don't you think that both the lines hold different meaning?

Reply:
@NitishKumar06 wouldn't it be better if deto localization for english is done for this line?

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
@bishal7679 That's what I also feel like! Adding text from our end might result in some errors. It'd be better if we follow the idea you mentioned. WDYT of this @rolfedh ?

Reply:
@rolfedh is not a native hindi speaker @NitishKumar06 😄

Reply:
@bishal7679 Are you able to open the slack? I'm not able to:(

Reply:
@bishal7679 @NitishKumar06 My idea while suggesting this change was to tell the readers that the Cluster architecture is concerned/associated with the best practices of high availability and security in a distributed system.

However I wasn't too sure on this suggestion, so I asked for other contributors for their thoughts before this suggestion was taken up. Thanks

Reply:
@Babapool Gotcha! I'm waiting for @rolfedh suggestion on the idea proposed by @bishal7679 regarding translating the deto English text to Hindi.

Reply:
> @bishal7679 That's what I also feel like! Adding text from our end might result in some errors. It'd be better if we follow the idea you mentioned. WDYT of this @rolfedh ?

Normally, you would be able to search for the id, `cluster-architect`, to see the glossary content in-context.
However, `dev-1.22-hi.1` doesn't have a topic that uses it.

Reply:
Glossary embeds in `*.md` files look like this:
```
* {{< glossary_tooltip text="Deployment" term_id="deployment" >}}
```

Reply:
Thanks @rolfedh

Reply:
Are there any more changes that need to be implemented now? @bishal7679 @Shivaniiiiiiiiiii